style.css

```diff
@@ -569,3 +567,114 @@ h4 {
   background: #fff;
   border-radius: 8px;
 }
+
+.form {
+  background-image: url('./images/form/illustration-contact-form.svg');
+  background-repeat: no-repeat;
+  background-size: 40px;
+  background-position: right bottom;
+  padding: 0;
+  margin-top: -850px;
+}
+
+.form-text {
+  grid-column: 1/13;
+  display: flex;
+  justify-content: center;
+  margin-left: 11px;
+  margin-bottom: 10px;
+}
+
+.form-text h3 {
+  font-family: 'Inter', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-size: 40px;
+  font-weight: 600;
+  line-height: 52px;
+  text-align: center;
+  margin-bottom: 35px;
+  color: #172b4d;
+}
+
+.inputs {
+  grid-column: 1/13;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.input {
+  width: 326px;
+  height: 40px;
+  border: 1px solid #d0d9d4;
+  border-radius: 4px;
+  background: #fff;
+  padding: 0;
+  margin-bottom: 30px;
+  margin-left: 15px;
+}
+
+.msg {
+  width: 326px;
+  height: 114px;
+  font-weight: 500;
+  border: 1px solid#60C095;
+  border-radius: 4px;
+  background: #fff;
+  padding: 0;
+  margin-bottom: 30px;
+  margin-left: 15px;
+}
```

Comment:
> - [ ] Please I suggest you use responsive values in your CSS rules, like percentages (i.e. width: 80%), instead of pixels. This allows your website to be more flexible and adaptable to different screen sizes and devices.😊